software-delivery-performance-terraform/sumologic_fer.tf

```diff
@@ -0,0 +1,81 @@
+#Sumo Logic SDP solution - This files has code to create FER's in Sumo Logic.
```

Comment:
Since this file is sumologic_fer, the other one should be sumologic_collection

Reply:
done